### PR TITLE
Check if bucket.creation_date is None to determine if bucket exists

### PR DESCRIPTION
**Pull request by mikkeloscar**
Behavior in boto3 seem to have been changed here:
https://github.com/boto/boto3/commit/60681e87

### CODE REVIEW

File: sevenseconds/config/s3.py [hunk 1]
```diff
@@ -40,9 +38,7 @@ def configure_s3_buckets(account: object):
 def create_logging_target(s3: object, logging_target: str, region: str):
     with ActionOnExit('Check logging target {}'.format(logging_target)) as act:
         logging_bucket = s3.Bucket(logging_target)
-        try:
-            logging_bucket.creation_date
-        except:
+        if not bucket.creation_date:
```
Comment (linki) [inline sevenseconds/config/s3.py]: should be `logging_bucket` I guess

Comment (mikkeloscar) [reply sevenseconds/config/s3.py]: :X Fixed.

**Comment by Raffo** [comment]
Build and test failing, on it.

**Comment by Raffo** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**